src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -156,12 +132,19 @@
             (.send xhr data)
             (.send xhr)))))))
 
+
 (defn ajax-orch [path arg-map & {:keys [service-prefix] :or {service-prefix "/api"}}]
   (assert (= (subs path 0 1) "/") (str "Path must start with '/': " path))
-  (ajax (assoc
-         arg-map :url (str (config/api-url-root) service-prefix path)
-         :headers (merge {"Authorization" (str "Bearer " @access-token)}
-                         (:headers arg-map)))))
+  (let [on-done (:on-done arg-map)]
+    (ajax (assoc
+           arg-map :url (str (config/api-url-root) service-prefix path)
+           :headers (merge {"Authorization" (str "Bearer " @access-token)}
+                           (:headers arg-map))
+           :on-done (fn [{:keys [status-code] :as m}]
+                      ;; Handle auth token expiration
+                      (if (= status-code 401)
```

Comment:
I don't like commenting on when a function should be used (I can't know all potential use cases), so I just updated the two non-orch call sites so if a developer is looking at any ajax call in our code, they either see `ajax-orch` or `ajax` with an explanation of why it is an exceptional case.